packages/did-ethr-resolver/src/utils/signers.ts

```diff
@@ -1,44 +1,6 @@
 /* eslint-disable guard-for-in */
 /* eslint-disable no-restricted-syntax */
 import { IKeys } from '@ew-did-registry/keys';
-import { ethers, Signer, providers } from 'ethers';
+import { ethers, Signer } from 'ethers';
 
 export const signerFromKeys = (keys: IKeys): Signer => new ethers.Wallet(keys.privateKey);
-
-export class ConnectedSigner extends ethers.Signer {
```

Comment:
Glad to see code removed 👍  . @knzeng-e would you mind just elaborating on why it was no longer needed please 🙇 ?

Reply:
As @JGiter explains me after this proposal, 

> Connected signer was needed because ethers v4 didn't allow first create Signer and later set provider on it. ethers v5 introduced Signer.connect(provider) for this purpose